selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/refactor/preview/PreviewRefactoringTest.java

```diff
@@ -134,19 +133,17 @@ public void checkPreviewRefreshTest() {
     refactorPanel.clickPreviewButtonRefactorForm();
     refactorPanel.clickOnItemByNameAndPosition("AppController.java", 0);
     loader.waitOnClosed();
-    refactorPanel.checkTextFromLeftEditor(EXPECTED_TEXT_BEFORE_CHANGE);
-    refactorPanel.checkTextFromRightEditor(EXPECTED_TEXT_IN_RIGHT_EDITOR);
+    refactorPanel.checkTextFromLeftEditor(EXPECTED_TEXT_IN_LEFT_EDITOR);
```

Comment:
Are the panels inverted in the new world?

Reply:
Compere widget for refactoring operation was not clear because the normal state of the compare widget is to have old version for left side and new version for the right side. So, yes I've switched the panels.